src/main/java/bio/terra/testrunner/runner/config/TestConfiguration.java

```diff
@@ -86,7 +86,7 @@ public static String readServerEnvironmentVariable() {
    * of the objects, for example by parsing the string values in the JSON object.
    */
   public void validate() {
-    logger.debug("Validating the server, Kubernetes and application specifications");
```

Comment:
ditto above: There's still an application specification on the test config, and we are still validating it below.

Reply:
Restored the original comments as well.